src/main/java/org/opentripplanner/updater/alerts/AlertsUpdateHandler.java

```diff
@@ -95,6 +95,12 @@ private void handleAlert(String id, GtfsRealtime.Alert alert) {
             if (informed.hasRouteId()) {
                 routeId = informed.getRouteId();
             }
+
+            String direction = null;
+            if (informed.hasTrip() && informed.getTrip().hasDirectionId()) {
+                direction = String.valueOf(informed.getTrip().getDirectionId());
```

Comment:
Well, in OBA GTFS modules it is stored as a string, so the conversion needs to take place somewhere, would it be more appropriate to do that in the apply-method instead when comparing the direction ids?